docs/ocf.md

```diff
@@ -13,13 +19,71 @@ protocol.
 
 Web IDL
 -------
-This IDL provides an overview of the interface; see below for documentation of
-specific API functions.
+This IDL provides an overview of the interfaces for OCF common, OCF Server and
+OCF Client; see below for documentation of specific API functions.
+
+The OCF Object
+--------------
+The OCF object is the top level object containing either OCF Server,
+OCF client, or both, as well as device and platform information.
 
 ```javascript
 // require returns an OCFObject
 // var ocf = require('ocf');
 
+interface OCFObject {
+    Server server;         // OCF server object
+    Client client;         // OCF client object
+    Platform platform;     // OCF platform info
+    Device device          // OCF device info
+};
+
+dictionary Platform {
+    string id;
+    string osVersion;
+    string model;
+    string manufacturerName;
+    string manufacturerURL;
+    string manufacturerDate;
+    string platformVersion;
+    string firmwareVersion;
+    string supportURL;
+}
+
+dictionary Device {
+    string uuid;
+    string name;
+    string dataModels;
+    string coreSpecVersion;
+}
+
+```
+The OCF device and platform objects can be set up after requiring 'ocf'. An
+example of this can be found in [OCF Server sample](../samples/OcfServer.js).
+The properties are registered to the system (and available during discovery)
+once either `OCFServer.registerResource()` or `OCFClient.findResources()`
+are called.
+
+OCF API Documentation
+---------------------
+
+### OCFObject.setBleAddress
+`void setBleAddress(string address);`
+
+Sets the devices BLE MAC address. This function is only defined on
+Zephyr boards with BLE capabilities (e.g. Arduino 101).
+
+The `address` parameter should be a MAC address string in the format
+`XX:XX:XX:XX:XX:X` where each character is in HEX format (0-9, A-F).
```

Comment:
Last section of sample format is missing an X.

Are lowercase a-f okay too, maybe list them?